miio/tests/test_vacuums.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022
+# Author: 2pirko
+import collections
+import sys
+from typing import List, Sequence, Tuple, Type
+
+import pytest
+
+from miio.device import Device
+from miio.integrations.vacuum.roborock.vacuum import ROCKROBO_V1
+from miio.interfaces import VacuumInterface
+
+from .test_device import DEVICE_CLASSES
+
+# list of all supported vacuum classes
+VACUUM_CLASSES: Tuple[Type[VacuumInterface], ...] = tuple(
+    cl for cl in DEVICE_CLASSES if issubclass(cl, VacuumInterface)  # type: ignore
+)
```

Comment:
Maybe `VacuumInterface.__subclasses__()` which should work for the time being until we find a better solution (which will be needed if/when the imports are removed from the main miio package)?